### DISCUSSION TITLE
NP & VI as data field options

### DISCUSSION THREAD

**Discussion by suskozaver** [SauceLLC/sauce4zwift#26] [Ideas]
Would it be possible to add NP and VI? 

I see it useful for races.

**Comment by mayfield** [comment]
@suskozaver NP is already in a few places, where specifically were you interested in seeing it?

Adding VI shouldn't be too difficult.  #29

**Reply by suskozaver** [reply]
just seeing it helped 🤦‍♂️ 
apology, i completely missed it! no idea how... maybe when I was clicking thru so many fields... 

great info for VI!
